frontend/src/infra/task/getTasks.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiClient } from '@/infra/lib/apiClient'
+import { Task } from '@/domain/Task'
+import { camelizeKeys } from '../lib/camelizeKeys'
+
+export const getTasks = async (
+  apiClient: ApiClient,
+  { userId, start, end }: { userId: number; start: string; end: string }
+): Promise<Array<Task>> => {
+  const params = new URLSearchParams({ user_id: userId.toString(), start, end })
+  const response = await apiClient(`/v1/timelog/tasks/?${params}`)
+
+  if (!response.ok) {
+    throw new Error('Failed to fetch Tasks')
+  }
+
+  const data = await response.json()
+
+  return camelizeKeys(data)
```

Comment:
Why do we need this method?

Reply:
because the api sends the json properties in snake_case, this transforms to camelCase

```
{
  project_id: 0
}

// becames

{
  projectId: 0
}
```

Reply:
Isn't it a bit overhead? We can send from the API in camel case already if that's better